### PR TITLE
Use repo to create tables for mds-db tests

### PR DESCRIPTION
**Pull request by drtyh2o**
## 📚 Purpose
*[description/approach]*

## 👌 Resolves:
- [ ] 🐛 JRA-834 fixes bug
- [ ] ✨ JRA-756 implements new feature

## 📦 Impacts:
*[list of packages]*

## ✅ PR Checklist
- [ ] Add or remove checklist items to suit your needs

### CODE REVIEW

File: packages/mds-db/migration.ts [hunk 1]
```diff
@@ -1,37 +1,24 @@
...
+import schema, { COLUMN_NAME, MANAGED_TABLE_NAME } from './schema'
 import { SqlExecuter, MDSPostgresClient } from './sql-utils'
 
 // drop tables from a list of table names
 async function dropTables(client: MDSPostgresClient) {
   const exec = SqlExecuter(client)
-  const drop = csv(schema.DEPRECATED_TABLES.concat(schema.TABLES))
+  const drop = csv(schema.DEPRECATED_TABLES.concat(schema.MANAGED_TABLES))
   await exec(`DROP TABLE IF EXISTS ${drop};`)
+  await Promise.all([GeographyRepository.revertAllMigrations(), PolicyRepository.revertAllMigrations()])
```
Comment (avatarneil) [inline packages/mds-db/migration.ts:14]: If we're already dropping these tables, won't the migration reverts barf?

File: packages/mds-db/tests/mds-db.spec.ts [hunk 2]
```diff
@@ -155,10 +157,9 @@ if (pg_info.database) {
 
       it('can make a successful read query after shutting down a DB client', async () => {
         await MDSDBPostgres.initialize()
```
Comment (avatarneil) [inline packages/mds-db/tests/mds-db.spec.ts:159]: wanna use the new `initializeDB` here, yeah?

**Review by avatarneil — COMMENTED** [review]
few questions

File: packages/mds-db/migration.ts (hunk 1 above)
Comment (drtyh2o) [reply packages/mds-db/migration.ts:14]: They aren't being dropped. They are excluded from the `DROP TABLE` command.

**Pull request merged**